### ISSUE TITLE
RESP protocol spec - typo in RESP Arrays example

### ISSUE THREAD

**Issue by brianseitel** [redis/redis-doc#1871]
I believe the docs on RESP Arrays have an incorrect example. The "hello world" example is listed as follows:

`"*2\r\n$3\r\nhello\r\n$3\r\nworld\r\n"`

If I break them out into newlines for readability, you get this:

```
*2 <-- an array of 2 entries
$3 <-- first entry: bulk string with length of 3
hello <-- string
$3 <-- second entry: bulk string with length of 3
world <-- string
```

However, the character lengths of `hello` and `world` would be 5, not 3, so the correct example should be:

`*2\r\n$5\r\nhello\r\n$5\r\nworld\r\n`

```
*2 <-- an array of 2 entries
$5 <-- first entry: bulk string with length of 5
hello <-- string (5 characters)
$5 <-- second entry: bulk string with length of 5
world <-- string (5 characters)
```

I tested this by setting up my on TCP listener on port 6379 that simply prints the inbound request, then ran `redis-cli hello world`, and my TCP repeater printed out `*2\r\n$5\r\nhello\r\n$5\r\nworld\r\n`, confirming my suspicions.

Hope this helps. Thanks.

**Comment by enjoy-binbin** [comment]
I didn't look at it closely, but it seems to be related to this one: https://github.com/redis/redis-doc/pull/1839 ?

**Comment by itamarhaber** [comment]
Thanks @brianseitel - looks like this has been resolved via #1879 so closing.
